pkgs/by-name/zm/zmkBATx/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  qt6,
+  pkg-config,
+  dbus,
+  simpleBluez,
+  simpleDBus,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "zmkBATx";
+
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "mh4x0f";
+    repo = "zmkBATx";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-xbiwRHVTuaZDH3RZlMK2CpKBThtS8g6q5r3C+OccDZg=";
+  };
+
+  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    pkg-config
+    qt6.qmake
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtconnectivity
+    dbus.lib
+    simpleBluez
+    simpleDBus
+  ];
+
+  configureFlags = [ "PREFIX=$out" ];
+
+  patchPhase = ''
```

Comment:
How about using `postPatch` ?

Reply:
Don't have strong opinion about this, done.